src/components/new-request/search.vue

```diff
@@ -44,8 +44,24 @@
       <!-- Change Name flow -->
       <template v-else-if="isChangeName">
         <BusinessLookupFetch />
-        <Jurisdiction v-if="isChangeNameXpro" />
-        <CompanyType v-if="showCompanyTypeRadioButtons" />
+        <Jurisdiction v-if="isChangeNameXpro" md="4"/>
+        <CompanyType v-if="getEntityTypeCd && isNumberedEntityType" />
+        <NumberedCompanyBullets v-if="isNumberedCompany" />
+        <template v-if="isNamedCompany && !isFederal">
+          <v-col cols="12" :md="showDesignation || isChangeNameXpro ? '8' : '12'">
+            <NameInput
+              :is-mras-search="(isXproFlow && isMrasJurisdiction && !getHasNoCorpNum)"
+              @emit-corp-num-validity="corpNumValid = $event"
+            />
+          </v-col>
+          <Designation v-if="showDesignation" cols="12" md="4" />
+        </template>
+        <XproFederalBullets v-if="isXproFlow && isFederal" cols="12" />
+        <v-row>
```

Comment:
You should not need this row here. We are already in a row. Everything here should be columns.